docs/remotely-debuggable-browsers.md

```diff
@@ -29,6 +29,7 @@ These are the flags necessary to start the remote debug server and use an altern
 * debug server `--start-debugger-server 6080`
 * separate profile `-P development` (not required)
  * **Note**: if you are prompted with the profile manager you will need to create a profile named `development`
+ * **Note**: in windows replace `--start-debugger-server 6080` with `-start-debugger-server 6080`
```

Comment:
Here's a TOC that might make it easier to find everything in this doc.  I don't think the links are all working, but its close if you want to include it.


#### Table Of Contents

[Firefox](#firefox)
  * [MacOS](#macos)
    * [Firefox Release](#firefoxrelease)
    * [Firefox Beta](#firefox-beta)
    * [Firefox Developer Edition](#firefox-developer-edition)
    * [Firefox Nightly](#firefox-nightly)
  * [Windows](#windows)
    * [Firefox (all versions)](#firefox-all-versions)
    * [Firefox (64 bit)](#firefox-64-bit)
  * [Android](#android)

[Chrome](#chrome)
  * [MacOS](#macos)
    * [Chrome (release)](#chromerelease)
    * [Chrome Canary](#chrome-canary)
  * [Windows](#windows)
    * [Chrome (all versions)](#chromeallversions)
    * [Chrome (64 bit)](#chrome-64-bit)